Library/Homebrew/caveats.rb

```diff
@@ -56,6 +56,9 @@ def keg_only_text(skip_reason: false)
       <<~EOS
         #{f.name} is keg-only, which means it was not symlinked into #{HOMEBREW_PREFIX},
         because #{f.keg_only_reason.to_s.chomp}.
+        If #{f.name} is a dependency library for another package you are
+        installing, the correct PATH's were used internally and no modification is needed.
```

Comment:
```suggestion
        installing, the correct PATHs were used internally and no modification is needed.
```

Reply:
That would be an even better way I think (if it's a dependency, don't output any caveats, or at least don't output them again at the end...)

Reply:
@rdp Great, glad we agree. Is that something you'd like/be able to tackle or shall I?

Reply:
Might be a bit involved for me...if you could that'd be great... :)